### PR TITLE
Download VM Demo

### PR DESCRIPTION
**Pull request by hosh**
References: Issue #7

### CODE REVIEW

**Comment by hosh** [comment]
What else is there? Need a check to make sure the ipfs daemon is running?

File: 001-download-vm/Makefile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+vm_image=QmW9on9UeKCVDb5dsTEZJQwQoH53gXUtVfWvSBhf2Thmyq/vms/opscode_ubuntu-14.04_chef-provisionerless.box
+
+download:
+	$(IPFS_BIN_PATH)/ipfs get $(vm_image)
```
Comment (jbenet) [inline 001-download-vm/Makefile:5]: want the output in the console videos to be super simple. let's use `ipfs` and maybe add a note that users should invoke `make` with a `$PATH` that includes `ipfs` ?

**Comment by jbenet** [comment]
other than comment above, this should work. Oh i guess we need:
- the uploading part? showing how to upload a VM
- the (human) script? i.e. what to say and do. e.g.

**Comment by hosh** [comment]
Oh crap, didn't see these comments. Will open a new one

**Comment by jbenet** [comment]
no worries! sgtm

**Pull request merged**